### PR TITLE
[FIX] l10n_it_edi: customer outside EU

### PR DESCRIPTION
**Pull request by nim-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by nim-odoo** [comment]
@jco-odoo any opinion?

File: addons/l10n_it_edi/data/invoice_it_template.xml [hunk 1]
```diff
@@ -107,13 +107,18 @@
                     </RappresentanteFiscale>
                     <CessionarioCommittente>
                         <DatiAnagrafici>
-                            <IdFiscaleIVA t-if="record.commercial_partner_id.vat">
+                            <IdFiscaleIVA t-if="record.commercial_partner_id.vat and in_eu(record.commercial_partner_id)">
                                 <IdPaese t-esc="get_vat_country(record.commercial_partner_id.vat)"/>
                                 <IdCodice t-esc="get_vat_number(record.commercial_partner_id.vat)"/>
                             </IdFiscaleIVA>
-                            <IdFiscaleIVA t-if="not record.commercial_partner_id.vat and record.commercial_partner_id.country_id.code != 'IT'">
+                            <IdFiscaleIVA t-if="not record.commercial_partner_id.vat and record.commercial_partner_id.country_id.code != 'IT' and in_eu(record.commercial_partner_id)">
```
Comment (jco-odoo) [inline addons/l10n_it_edi/data/invoice_it_template.xml:114]: I think the condition as before was correct as it should be the same for private clients as well.  I am also not sure it should have the IdPaese.  If there was no problem with that before.

File: addons/l10n_it_edi/data/invoice_it_template.xml [hunk 2]
```diff
@@ -107,13 +107,18 @@
...
+                                <IdPaese t-esc="record.commercial_partner_id.country_id.code"/>
                                 <IdCodice t-esc="'0000000'"/>
                             </IdFiscaleIVA>
+                            <IdFiscaleIVA t-if="not in_eu(record.commercial_partner_id)">
```
Comment (jco-odoo) [inline addons/l10n_it_edi/data/invoice_it_template.xml:118]: only when you have a VAT record

**Review by jco-odoo — CHANGES_REQUESTED** [review]
@nim-odoo I think the logic is not entirely correct for the private customers.

**Comment by nim-odoo** [comment]
@jco-odoo corrected, thanks

**Comment by nim-odoo** [comment]
robodoo r+

**Pull request closed without merging**